admin-dev/themes/default/template/controllers/groups/helpers/tree/tree_categories.tpl

```diff
@@ -82,7 +82,7 @@
 		$("#{$id|escape:'html':'UTF-8'}").tree("collapseAll");
 
 		{if isset($selected_categories)}
-			{assign var=imploded_selected_categories value='","'|implode:$selected_categories}
```

Comment:
The code is not equivalent before and after. We must use `implode('","', $selected_categories);`
Nobady saw it 😄